mobile/lib/repositories/user.repository.dart

```diff
@@ -1,17 +1,23 @@
 import 'package:http/http.dart' as http;
 import 'package:triggo/api/call.dart';
 import 'package:triggo/api/response.dart';
+import 'package:triggo/repositories/credentials.repository.dart';
 import 'package:triggo/repositories/dtos/user.dtos.dart';
 
 class UserRepository {
   final http.Client? client;
+  final CredentialsRepository? credentialsRepository;
 
-  UserRepository({this.client});
+  UserRepository({this.client, required this.credentialsRepository});
 
   Future<Response<OutGetUserDTO>> getUser() async {
     final response = await call(
       method: 'GET',
       endpoint: '/user',
+      headers: {
+        'Authorization':
+            'Bearer ${await credentialsRepository!.getAccessToken()}',
+      },
       client: client,
     );
```

Comment:
Just my opinion but :
```suggestion
    final accessToken = await credentialsRepository.getAccessToken();
    final response = await call(
      method: 'GET',
      endpoint: '/user',
      headers: {
        'Authorization':
            'Bearer ${accessToken}',
      },
      client: client,
    );
```

Reply:
Didn't think about it, you're right